src/utils/update-status-comment.ts

```diff
@@ -34,19 +37,21 @@ export const updateStatusComment = async ({
       (comment.body ?? "").indexOf(METICULOUS_COMMENT_IDENTIFIER) > -1
   );
 
+  const fullBody = `${body}[^1]\n\n[^1]:Last updated for commit ${shortHeadSha}. This comment will update as new commits are pushed.${METICULOUS_COMMENT_IDENTIFIER}}`;
```

Comment:
Ideally this would be `[^1]: Last updated at 6:15pm for commit 6e2856e.`, but to not be confusing we'd want it converted to the user's time zone, and that's not really possible for a GH comment (no dynamic rendering, different users may be in different time zones, and (I suspect) no way to discover from the action triggered by GH BE what time zone the majority of users of the repo are in). 

You do however get the time at the top of the comment:

![image](https://user-images.githubusercontent.com/793481/221155573-b345619f-bbcc-4c11-9195-5ad2e8ae4257.png)